cmake/FindMKL.cmake

```diff
@@ -0,0 +1,122 @@
+# NOTE: it is not an official cmake search file
+# source: https://gist.github.com/scivision/5108cf6ab1515f581a84cd9ad1ef72aa
+#
+################################################################################
+# NOTE: this file was originally developer by:
+#
+# \file      cmake/FindMKL.cmake
+# \author    J. Bakosi
+# \copyright 2012-2015, Jozsef Bakosi, 2016, Los Alamos National Security, LLC.
+# \brief     Find the Math Kernel Library from Intel
+# \date      Thu 26 Jan 2017 02:05:50 PM MST
+#
```

Comment:
Can we include this? What's the license of the original file?

Reply:
The author said that it was BSD-3 license